emulator/iSkipper/iClickerEmulator.h

```diff
@@ -49,7 +46,8 @@ const uint8_t answerOffsets[NUM_ANSWER_CHOICES] = {0x1, 0x5, 0xd, 0xe, 0xa, 0x2}
 const uint8_t DEFAULT_SEND_SYNC_ADDR[SEND_SYNC_ADDR_LEN] =
     {RF_SYNC_BYTE1_VALUE_IC, RF_SYNC_BYTE2_VALUE_IC , RF_SYNC_BYTE3_VALUE_IC };
 
-const uint8_t ACK_SYNC_ADDR[ACK_HEADER_SIZE] = {0x55, 0x55, 0x55};
+#define ACK_SEND_SYNC_ADDR_LEN 3
```

Comment:
`ACK_SEND_SYNC_ADDR_LEN` is the same as `RECV_SYNC_ADDR_LEN`. `ACK_SEND_SYNC_ADDR_LEN` should be removed and `RECV_SYNC_ADDR_LEN` should be set to 3. The radio sets the sync address length based off of `RECV_SYNC_ADDR_LEN`.